pgmetadata/controllers/dcat.classic.php

```diff
@@ -139,19 +134,16 @@ public function index()
             if (!empty($query)) {
                 if (!$this->isValidQueryString($query)) {
                     // We use a fake UID to get no data
-                    $option = 'get_rdf_dcat_catalog_by_id';
                     $fake_id = 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11';
-                    $filterParams[] = $fake_id;
+                    $result = $search->getRDFDcatCatalogById($locale, $fake_id, $profile);
                 } else {
-                    $option = 'get_rdf_dcat_catalog_by_query';
-                    $filterParams[] = $query;
+                    $result = $search->getRDFDcatCatalogByQuery($locale, $query, $profile);
                 }
+            } else {
+                $result = $search->getRDFDcatCatalog($locale, $profile);
             }
         }
```

Comment:
```suggestion
            } else {
                $result = $search->getRDFDcatCatalog($locale, $profile);
            }
        } else {
            $result = $search->getRDFDcatCatalog($locale, $profile);
        }
```

Reply:
you cannot do that, else you have two `else`. See line 129.
```
} else {
      $result = $search->getRDFDcatCatalog($locale, $profile);
}
```
it is already into a ` } else {`.

Reply:
I fully agree. I must have opened the file in another FF tab, instead of reading the side-by-side diff.. Sorry